crates/sui-indexer-alt-consistent-store/src/rpc/consistent_service/available_range.rs

```diff
@@ -9,12 +9,16 @@ use super::State;
 
 pub(super) fn available_range(
```

Comment:
Not sure if this is documented somewhere else but some docs here could be helpful in understanding what the safe reading range is. 

I got tripped up thinking about `min_checkpoint: range.as_ref().map(|r| r.start().checkpoint_hi_inclusive)`. Is it because we have a snapshot for every checkpoint (`stride = 1`) so the `checkpoint_hi_inclusive` of the starting snapshot is the safe "reader_lo"?

Reply:
Sorry raced on this thread with you @wlmyng. In case it matters:

> if each column family is configured with a different retention + stride we'd also want to eventually expose that, like a per-entity and per-table min amd max checkpoint

We can't set different retentions and strides per pipeline/column family, because snapshots are a database-wide concept.

Reply:
I've added some more docs on `AvailableRangeResponse` describing how `stride` works, hopefully that helps to clear things, but let me know if not!

Reply:
Thanks @emmazzz that's what I mean't. This comment
```
Accessible checkpoints in the available range are evenly spaced out
    /// `stride` checkpoints apart.
```
helped clarify things for me!